src/commands/test.rs

```diff
@@ -48,53 +48,17 @@ struct Transaction {
 }
 
 #[derive(Debug, Serialize, Deserialize)]
-#[serde(tag = "type")]
-enum Expect {
-    Balance(ExpectBalance),
-    // TODO: improve expect adding more options
+struct ExpectUtxo {
+    from: String,
+    datum_equals: Option<serde_json::Value>,
+    min_amount: Option<Vec<ExpectMinAmount>>,
```

Comment:
no need for `Option<Vec<>>`, just use `Vec<>` and consider the empty vec as the None